### DISCUSSION TITLE
FT, NFT 多平台发布管理平台

### DISCUSSION THREAD

**Discussion by anthonyhuanggr** [rebase-network/hello-world#47] [Ideas]
hi Rebase社区的大神们，

就是想搞个FT,NFT的多平台发布交易管理的系统。
FT, NFT 多的话，多平台转来转去比较麻烦，如果实现统一管理就好。
望指点，谢谢

**Comment by luckyyang** [comment]
可以啊，你现在遇到什么技术问题了吗，可以具体说说
